var/exp_repo/experiments/laghos/experiment.py

```diff
@@ -0,0 +1,62 @@
+from benchpark.directives import variant
+from benchpark.experiment import Experiment
+
+
+class Laghos(Experiment):
+
+    variant(
+        "experiment",
+        default="example",
+        values=("weak", "strong","example"),
+        description="weak or strong scaling",
+    )
+
+    def compute_applications_section(self):
+        variables = {}
+            
+        if self.spec.satisfies("scaling=weak"):
```

Comment:
Let's start with the scaling assuming a single node, but change problem size per rank instead, once we know we have this right we can add more nodes or make updates. So for weak scaling:
rs values: 1
num processes: 8
rp values (increases workload per rank): 1,2,3,4

Another option if you can also implement this is rs values 1,2,3 and mpi processes 1,8,64